tests/test_module_alascan.py

```diff
@@ -407,7 +410,10 @@ def test_alascan_cluster_analysis(protprot_input_list, scan_file):
 
     # clean up
```

Comment:
Just in general, if your assert comes before the cleaning, if the test fail the files will not be deleted and this might interfere with the next round of tests (causing some artifact)

Reply:
yep, that's a problem..should use the tempfile module